projects/igniteui-angular/src/lib/core/styles/components/grid/_grid-theme.scss

```diff
@@ -1192,6 +1192,21 @@
         align-items: center;
         outline-style: none;
 
+        @extend %cell-input-overrides;
```

Comment:
Since you are extending this here, you can remove that same extend from `igx-grid-td--editing` since the base class `igx-grid_td` is always present